### PR TITLE
Manifests: installer updates

### PR DESCRIPTION
**Pull request by squeed**
We need to tweak the installer manifests slightly:

* Need to be a DaemonSet
* Need locking
* Rename the files a bit

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_out
 COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-renderer /bin/cluster-network-renderer
 COPY manifests /manifests
 COPY bindata /bindata
+COPY testbindata /testbindata
```
Comment (danwinship) [inline Dockerfile]: We really want test data in the image? Is that what other people are doing?

Comment (squeed) [reply Dockerfile]: I'm not sure. I haven't really decided how to do e2e testing. I could get rid of this you like.

Comment (danwinship) [reply Dockerfile]: I don't really know...

**Comment by squeed** [comment]
Bah, it looks like I'll have to completely refactor the operator to pick up a small change to the operator-sdk leaderelection logic.

Gonna close this PR, as the new one will be quite different.

**Pull request closed without merging**